packages/querygrid/package.json

```diff
@@ -39,7 +39,7 @@
   },
   "homepage": "https://github.com/LabKey/glass-components#readme",
   "dependencies": {
-    "@glass/base": "0.21.0",
+    "@glass/base": "0.21.0-19.3-fb-uxddPolish2.1",
     "@glass/domainproperties": "0.0.48",
```

Comment:
Since both base and domainproperties package versions have updates, this querygrid version should be updated as well and both of these dependencies should update. This is likely the reason that the yarn.lock file has @glass references below.

Also, don't forget a querygrid/README update for this.

Reply:
Ok cool thanks.  I knew we didn't want that in yarn.lock just wasn't sure how it got there.